hack/tools/tools.go

```diff
@@ -1,4 +1,3 @@
-//go:build tools
 // +build tools
```

Comment:
I'm not sure what these directives do. Are they for bazel? If so don't we still need them?

Reply:
These are not for bazel. gofmt enforces `//go:build tools` over `// +build tools` starting 1.17, and this change was an accident from the first commit, in current master there is only `// +build tools`